### ISSUE TITLE
Error parsing github actions workflow

### ISSUE THREAD

**Issue by mcpherrinm** [woodruffw/zizmor#64]
I tried to run zizmor on https://github.com/letsencrypt/boulder/blob/main/.github/workflows/boulder-ci.yml

I got the following error:

```
% zizmor .
Error: invalid GitHub Actions workflow: "/src/boulder/.github/workflows/boulder-ci.yml"

Caused by:
    jobs: data did not match any variant of untagged enum Job at line 28 column 3
```

I believe this is job is valid as it works. Without reading the zizmor source, I'm not sure what enum it's expecting to match. Line 28 col 3 is the name of the job.

**Comment by woodruffw** [comment]
Thanks @mcpherrinm! That's almost certainly a bug in https://github.com/woodruffw/github-actions-models -- I'll debug.

**Comment by woodruffw** [comment]
https://github.com/woodruffw/github-actions-models/pull/8 will fix this -- the underlying problem is that `strategy` doesn't require a `matrix`, even though the JSON Schema for GitHub Workflows ([source](https://json.schemastore.org/github-workflow.json)) implies it does.

**Comment by woodruffw** [comment]
Bump and full fix is in #65, and I've confirmed it works for me locally. Thanks for the report @mcpherrinm!

**Comment by mcpherrinm** [comment]
Thanks!  zizmor now successfully parses all the repos I have locally at least!
